src/components/Comments.jsx

```diff
@@ -1,12 +1,15 @@
+import { useState } from "react";
 import useComments from "../hooks/useComments";
 import Loading from "./Loading";
 
 const Comments = ({id}) => {
-    const { commentsData, isLoading } = useComments(id); 
+    const [limit, setLimit] = useState(null);
+    const { commentsData, totalComments, isLoading } = useComments(id, limit);
+    const [isHidden, setIsHidden] = useState(false);
 
     return <section className="article-comments">
             <h3>Comments</h3>
-            {isLoading ? <Loading></Loading> : <ul className="comments-list">
+            {isLoading ? <Loading></Loading> : isHidden ? null : <ul className="comments-list">
```

Comment:
Just need to add some consideration for an article that has no comments. A message to the user lets them know that there are no comments, rather than them potentially thinking something went wrong fetching them